src/html/partials/header.html

```diff
@@ -0,0 +1,14 @@
+<header class="page__header">
+  <div class="wrapper--title">
+    <div class="page__header--left">
+      <img src="" alt="">
+    </div>
+    <h1 class="page__header--title">A Viget Exploration</h1>
+    <div class="page__header--right">
+      <img src="" alt="">
+    </div>
+  </div>
+  <div class=hero__image">
```

Comment:
Similar BEM observation here, since there isn't a `hero` class above this element, you don't need to specify both block and element here (e.g. `hero` and `image`), it could just be `hero-image` or just `hero`.